cfg/cis-1.6-k3s/master.yaml

```diff
@@ -0,0 +1,872 @@
+---
+controls:
+version: "cis-1.6-k3s"
+id: 1
+text: "Master Node Security Configuration"
+type: "master"
+groups:
+  - id: 1.1
+    text: "Master Node Configuration Files"
+    checks:
+      - id: 1.1.1
+        text: "Ensure that the API server pod specification file permissions are set to 644 or more restrictive (Not Applicable)"
+        scored: false
+
+      - id: 1.1.2
+        text: "Ensure that the API server pod specification file ownership is set to root:root (Not Applicable)"
+        scored: false
+
+      - id: 1.1.3
+        text: "Ensure that the controller manager pod specification file permissions are set to 644 or more restrictive (Not Applicable)"
+        scored: false
+
+      - id: 1.1.4
+        text: "Ensure that the controller manager pod specification file ownership is set to root:root (Not Applicable)"
+        scored: false
+
+      - id: 1.1.5
+        text: "Ensure that the scheduler pod specification file permissions are set to 644 or more restrictive (Not Applicable)"
+        scored: false
+
+      - id: 1.1.6
+        text: "Ensure that the scheduler pod specification file ownership is set to root:root (Not Applicable)"
+        scored: false
+
+      - id: 1.1.7
+        text: "Ensure that the etcd pod specification file permissions are set to 644 or more restrictive (Not Applicable)"
+        scored: false
+
+      - id: 1.1.8
+        text: "Ensure that the etcd pod specification file ownership is set to root:root (Not Applicable)"
+        scored: false
+
+      - id: 1.1.9
+        text: "Ensure that the Container Network Interface file permissions are set to 644 or more restrictive (Not Applicable)"
+        scored: false
+
+      - id: 1.1.10
+        text: "Ensure that the Container Network Interface file ownership is set to root:root (Not Applicable)"
+        scored: false
+
+      - id: 1.1.11
+        text: "Ensure that the etcd data directory permissions are set to 700 or more restrictive if etcd is used (Automated)"
+        audit: if [ -d "/var/lib/rancher/k3s/server/db/etcd" ];then stat -c permissions=%a /var/lib/rancher/k3s/server/db/etcd; else echo 'NotEtcd'; fi
```

Comment:
> if etcd is used
> 'NotEtcd'

I can't find the source from k3s document for this case.

Reply:
It's from https://docs.rancher.cn/docs/k3s/security/self-assessment/_index/ of 1.1.11. 

> if etcd is used 'NotEtcd'

This is my idea, If the path `/var/lib/rancher/k3s/server/db/etcd` not found, take it pass to prevent false positives, because not the test case false, it's only the file path cannot be found.

Reply:
What if the k3s server use another directory as etcd data directory? In this case, it should fail instead of pass.
IMHO, we should not allow the check to be passed when the file path cannot be found.